test/proxy.test.js

```diff
@@ -0,0 +1,415 @@
+'use strict';
+
+const test = require('tape');
```

Comment:
This test file looks like it was copied from `test/build.test.js` and modified. I presume you did this because it is different enough that it does not make sense to integrate into `test/build.test.js`? If so, can you at least clean up the commented code in this file to avoid future confusion of whether that commented code is intended to work?

Reply:
i am happy to remove the commented out code; the reason i left it there was to keep the code as parallel as possible for future comparison. if that doesn't seem worthwhile to you then i'll clean it up.

my theory was that testing the proxy with a single test app was good enough; if that is a bad assumption i can just uncomment the additional tests and work on merging this file into `build.test.js`.

Reply:
@springmeyer - this is a bit of an open question for me. the other items are cleared.

Reply:
👍  seems fine to test with just one app. So, seems okay to just remove the commented code and simplify the test code as much as possible.

Reply:
removed